### PR TITLE
New release notification

### PR DESCRIPTION
**Pull request by ShGKme**
### ☑️ Resolves

* Issue #84 

### 💁 Description

Every 2h checks if there is a new release on GitHub releases and notify with a push notification. Click on the notification opens the latest release page.

### 🖼️ Screenshots

image

### CODE REVIEW

**Review by nickvergessen — APPROVED** [review]
I modified it to check the freshly created release org repo

File: src/app/githubReleaseNotification.service.js [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+		clearInterval(schedulerIntervalId)
+	}
+}
+
+module.exports = {
+	getLatestReleaseVersion,
+	notifyAboutNewVersion,
+	checkForNewVersion,
+	setupReleaseNotificationScheduler,
+	stopReleaseNotificationScheduler,
```
Comment (nickvergessen) [inline src/app/githubReleaseNotification.service.js:126]: grafik


PS most of them seem to not be used outside, do you export by default? Or does it make sense to not export it?

**Comment by ShGKme** [comment]
@nickvergessen Before merging, do you think ~2h interval is ok?

**Comment by nickvergessen** [comment]
yes, would have even opted for "in the morning only"

**Comment by ShGKme** [comment]
> yes, would have even opted for "in the morning only"

User settings are planned for later this week.

**Pull request merged**